resources/windows/wix/condition.wxs.erb

```diff
@@ -18,11 +18,10 @@
         <![CDATA[Installed OR (VersionNT >= 600)]]>
       </Condition>
 
-      <!--
       <Condition Message="32 bit <%= settings[:product_name] %> is no longer supported on 64 bit Windows Operating Systems.">
-          <![CDATA[Installed OR VersionNT64]]>
+          <![CDATA[Installed OR Not VersionNT64]]>
```

Comment:
@Iristyle Had to change the ```OR VersionNT64``` to ```OR Not VersionNT64``` as when I first tested it, the logic was incorrect - i.e. it blocked the 32bit install on a 32 bit install instead of allowing it.